manuscript/070_Practicing_What_We_Already_Learned.md

```diff
@@ -736,7 +736,7 @@ public class Calculator
 
 **Benjamin:** Wow, looks like a lot of code just to make the Statement false! Is it worth the hassle? We will undo this whole modification in a second anyway
 
-**Johnny:** Depends on how confident you want to feel. I would say that it is usually worth it -- at least you know that you got everything right. It might seem like a lot of work, but it actually took me about a minute to add this code and imagine you got it wrong and had to debug it on a production environment. \_That\_ would be a waste of time.
+**Johnny:** Depends on how confident you want to feel. I would say that it is usually worth it -- at least you know that you got everything right. It might seem like a lot of work, but it took me about a minute to add this code and imagine you got it wrong and had to debug it on a production environment. \_That\_ would be a waste of time.
```

Comment:
I have this feelings that If we delete "actually" here, we need to put something else in. The goal here was to emphasize that what "actually" happened, was different than what seemed it would happen. If I just take "actually" away, I don't feel that the "but" introduces enough emphasis on the contrast between what was supposed to happen and what happened.

I am not a native speaker though, so maybe it's just me...